Update add-to-app iOS arm64 simulator instructions (#6874)
flutter/website: src/development/add-to-app/ios/project-setup.md

## Apple Silicon (`arm64` Macs)

> On an Apple Silicon (M1) Mac, the host app will build for an `arm64` simulator.
> While Flutter does support `arm64` simulators, some plugins may not. If you use
> one of these plugins, you may see a compilation error like **Undefined symbols
> for architecture arm64** and you must exclude `arm64` from the simulator
> architectures in your host app.

Reviewer: ```suggestion
On an Apple Silicon (M1) Mac, the host app builds for an `arm64` simulator.
While Flutter supports `arm64` simulators, some plugins might not. If you use
one of these plugins, you might see a compilation error like **Undefined symbols
for architecture arm64** and you must exclude `arm64` from the simulator
architectures in your host app.
```
